### PR TITLE
Lines fragment mvp

### PR DESCRIPTION
**Pull request by maras93**

### CODE REVIEW

File: app/src/main/java/com/example/marekmacko/czyprzejade/models/Line.java [hunk 1]
```diff
@@ -117,4 +140,19 @@ public String getDate() {
...
+    public int describeContents() {
+        return 0;
+    }
+
+    @Override
+    public void writeToParcel(Parcel dest, int flags) {
+        dest.writeInt(id);
+        dest.writeString(line);
+        dest.writeString(url);
+        dest.writeInt(isChanged ? 1 : 0);
```
Comment (dawhey) [inline app/src/main/java/com/example/marekmacko/czyprzejade/models/Line.java]: Magic numbers - extract to variable FLAG_CHANGED/FLAG_UNCHANGED

File: app/src/main/java/com/example/marekmacko/czyprzejade/fragments/LineNumbersFragment.java [hunk 2]
```diff
@@ -48,35 +54,65 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
...
-
-            @Override
-            public void onFailure(RetrofitError error) {
-                // TODO: error handling
-                Toast.makeText(getContext(), "TramLines download failed :(", Toast.LENGTH_SHORT).show();
-            }
-        }));
+    @Override
+    protected String getErrorMessage(Throwable e, boolean pullToRefresh) {
+        return "Brak danych :(";
```
Comment (dawhey) [inline app/src/main/java/com/example/marekmacko/czyprzejade/fragments/LineNumbersFragment.java]: Hardcoded string - extract to string resource

File: app/src/main/java/com/example/marekmacko/czyprzejade/models/Line.java (hunk 1 above)
Comment (maras93) [inline app/src/main/java/com/example/marekmacko/czyprzejade/models/Line.java]: It's not a magic numbers but representation of boolean. We don't have method like writeBool so we must convert this to int.

File: app/src/main/java/com/example/marekmacko/czyprzejade/fragments/LineNumbersFragment.java (hunk 2 above)
Comment (maras93) [inline app/src/main/java/com/example/marekmacko/czyprzejade/fragments/LineNumbersFragment.java]: In final version toast will be deleted together with this message, so I don't see reason to extracting him.

**Pull request merged**